README.md

```diff
@@ -44,24 +46,54 @@ To grant access:
 ## Step 3: Clone the repo 
 Clone the repo to your development machine. 
 
-The project has two relevant Nuget packages:
-1. Microsoft.Azure.Services.AppAuthentication - makes it easy to fetch access tokens for Service-to-Azure-Service authentication scenarios. 
```

Comment:
The equivalent package would be Azure.Identity

Reply:
May as well add back the second line and include `Azure.Messaging.ServiceBus` instead.